### PR TITLE
Event stickies

### PR DESCRIPTION
**Pull request by Apogate**
This is a neat feature, allow admins to create self-pruning stickies. Once a thread reaches a certain number of posts, the oldest replies in the thread are deleted. Useful for running events + browsers that can't handle loading 5000 posts at once

Default config value (`EVENT_STICKY_RES`) is **`1500` posts**

goodnight team

### CODE REVIEW

**Comment by Yushe** [comment]
So new type of sticky thats cool, but are the stickies gonna be fixed? (stay at the top and won't fall down)

Edit: I have found out that older versions of saguaro stickies work.

**Comment by Apogate** [comment]
I re-tested this branch for that issue.
If you're still having problems with stickies, this will fix it.

File: config.php [hunk 1]
```diff
@@ -60,6 +60,9 @@
 define(BLOTTER_PATH, 'CHANGEME'); //Absolute html path to your blotter file, this feature is experimental and still is not fully functional.
 
 // Post & Thread
+define(EVENT_STICKY_RES, 1500); //The number of replies allowed to an event sticky before self-pruning begins.
```
Comment (RePod) [inline config.php]: Should be moved down to line 79/82 (next to _MAX_RES_) for consistency.

Comment does not need to be multiple lines either.

**Comment by Apogate** [comment]
Merging. wew

**Pull request merged**